doc/working-with-spyder.rst

```diff
@@ -22,3 +22,39 @@ Explore how to take advantage of Spyder’s functionality beyond just the four c
 * Quickly navigate within and between files with the Outline pane
 * Search for text or regular expressions across your entire project with the Find pane
 * Discover and explore structured documentation in the Online Help pane
+
+
+
+===========================
+Improving your code quality
+===========================
+
+Learn how to improve the quality of your programs using code analysis.
+
+.. youtube:: JnljnAjdO_w
+            :height: 360
+            :width: 640
+            :align: left
+
+* Open and use Code Analysis to evaluate the quality and style of Python files
+* Run analysis on a file in the Editor, on your computer or at a path
```

Comment:
```suggestion
* Run analysis on a file in the Editor or in your computer
```

Reply:
@CAM-Gerlach was the one who wrote it so I think "on" is right. I'll let him confirm this and adjust your suggestion accordingly.

Reply:
Ok, it's just that the current text is a bit complicated to understand.

Reply:
Yeah, I agree with removing the "or at a path" part.

Reply:
```suggestion
* Run analysis on a file in the Editor or anywhere on your computer
```

@juanis2112 is right, "on" is the correct word in this context. I also agree with simplifying it; I've added "anywhere" to help balance the phrasing

Reply:
Agreed. I was thinking to suggest something along those lines, but I couldn't come up with a sentence that sounded good to me.